terraform/010-cluster/main.tf

```
@@ -8,6 +8,18 @@ module "vpc" {
   aws_zones = var.aws_zones
 }
```

Each module needs a `providers` block to identify which zone should be used. [This](https://developer.hashicorp.com/terraform/language/modules/develop/providers) is a good resource.
```suggestion

    providers = {
    aws = aws.primary
  }
}
```